tools/clang/test/taef/lit.cfg

```diff
@@ -28,7 +28,7 @@ if llvm_obj_root is None:
 
 te = os.path.join(config.llvm_obj_root, "TAEF", config.arch, 'te.exe')
 
-test_dll = os.path.join(config.llvm_obj_root, config.build_mode, 'bin', 'clang-hlsl-tests.dll')
+test_dll = os.path.join(config.llvm_obj_root, config.build_mode, 'bin', 'ClangHLSLTests.dll')
```

Comment:
Out of belated curiosity, why the rename?

It looks like the previous DLL is still referenced by HLSLHost.cpp and a few powershell and cmd files. Less important, but possibly worth it for consistency, the gtest executable equivalent is named `clang-hlsl-tests` to keep with the previous naming convention.